blogposts/2022/release-3.36.md

```diff
@@ -0,0 +1,60 @@
+    url: https://github.com/sourcegraph/sourcegraph/pull/29856
+    category: Search
+  - description: 'Symbolic links are now searchable to make files easier to find. Previously it was possible to navigate to symbolic links in the repository tree view, however the symbolic links were ignored during searches.'
+    url: https://github.com/sourcegraph/zoekt/pull/237
+    category: Search
+  - description: "Sourcegraph's API (streaming search, GraphQL, etc.) may now be used from any domain when using an access token for authentication, or with no authentication in the case of Sourcegraph.com. This enables customers to use their Sourcegraph instance's APIs in new ways, such as calling them via web applications."
+    url: https://github.com/sourcegraph/sourcegraph/pull/28775
+    category: API
+  - description: 'The endpoint `/search/stream` will be retired in favor of `/.api/search/stream`. This requires no action unless you have developed custom code against `/search/stream`. We will support both endpoints for a short period of time before removing `/search/stream`. Please refer to the documentation for more information.'
+    url: https://docs.sourcegraph.com/api/stream_api
+    category: API
+---
+
+Sourcegraph 3.36 is now available! Here are some highlights from this release:
+
+## Push new branches from Batch Changes to forks
+
+Before this release, creating a batch change required you (or your service-account) to have write-access to every target repository. This was incompatible with having tight repository access control, and was impractical at a large scale. We heard from customers that they wanted all users to be able to create batch changes on all repositories, without them needing overly broad write permissions.
+
+That's why we are introducing [Batch Changes on forks](https://docs.sourcegraph.com/admin/config/batch_changes#forks), which makes it possible for branches created by Batch Changes to be pushed to a fork of the repository rather than the repository itself. This solves those access control challenges and lets you safely enable Batch Changes for everyone. In Sourcegraph 3.36, this is an instance setting that can be turned either on or off for all users, and is off by default.
+
+## Find repo files faster with fuzzy finder
+
+We're excited to introduce a new fuzzy file finder which can help you quickly open a file from your repository. The keyboard shortcut to activate it is Cmd+K on macOS, or Ctrl+K on Linux/Windows. Note that it will only activate when you're on repository-related pages, such as the repository overview page for example.
+
+<img class="blog-image" title="Fuzzy finder" alt="An example of using fuzzy finder to navigate to test files within the sourcegraph/sourcegraph repo." src="https://storage.googleapis.com/sourcegraph-assets/blog/3.36/fuzzyfinder.gif">
```

Comment:
@jyavorska Wondering if you know how to center this image?

<img width="1088" alt="Screenshot 2022-01-21 at 11 46 35" src="https://user-images.githubusercontent.com/82806852/150548270-174c1e5c-a610-4166-b757-6f179ebab8d8.png">